### DISCUSSION TITLE
Add option to set Legend width

### DISCUSSION THREAD

**Discussion by DanielLeone** [grafana/grafana#44504] [Dashboards, Panels & Visualizations]
**What would you like to be added**

In the new Time Series view, I can't **set the legend width**, so time series charts don't line up vertically very nicely:

![image](https://user-images.githubusercontent.com/8600684/151280207-1d831e53-c7aa-420c-88ae-f8f3b0245554.png)

In previous Grafana versions, or by using Graph (old), I can set the width of the legend so all the charts visually line up:

![image](https://user-images.githubusercontent.com/8600684/151280085-4b4a9404-f230-4e35-b5c6-8962a96f63d5.png)

**Why is this needed (describe your use case and goals):**

Having charts line up vertically makes it easy to correlate events between charts without having to use a shared cursor, like so:

![image](https://user-images.githubusercontent.com/8600684/151280529-f2166c5b-d165-4fd9-8fea-871f120b6827.png)

Also mentioned here: https://community.grafana.com/t/legend-width-in-the-new-time-series-visualization/53994

Happy to help implement when I get some time

**Comment by bobrik** [comment]
See my attempt at making this work as before: https://github.com/grafana/grafana/pull/49126.

**Comment by gelicia** [comment]
Looks like the PR for this was merged in #49126 Closing discussion
